hide/comp/cdb/ModalColumnForm.hx

```diff
@@ -0,0 +1,238 @@
+
+            <div>Column name</div> <input type="text" name="name"/>
+			<br /><br />
+
+            <div>Column type</div>
+            <select name="type" onchange="$(\'#col_options\').attr(\'class\',\'t_\'+this.value)">
+                <option value="">---- Choose -----</option>
+                <option value="id">Unique Identifier</option>
+                <option value="string">Text</option>
+                <option value="bool">Boolean</option>
+                <option value="int">Integer</option>
+                <option value="float">Float</option>
+                <option value="enum">Enumeration</option>
+                <option value="flags">Flags</option>
+                <option value="ref">Reference</option>
+                <option value="list">List</option>
+                <option value="properties">Properties</option>
+                <option value="color">Color</option>
+                <option value="file">File</option>
+                <option value="image">Image</option>
+                <option value="tilepos">Tile</option>
+                <option value="dynamic">Dynamic</option>
+                <option value="layer">Data Layer</option>
+                <option value="tilelayer">Tile Layer</option>
+                <option value="custom">Custom Type</option>
+            </select>
+            <br /><br />
+            
+			<div id="col_options">
+				<div class="values">
+                    Possible Values
+                    <br />
+                    <input type="text" name="values" name="vals"/>
+                    <br /><br />
+                </div> 
+
+				<div class="sheet">
+                    Sheet
+                    <br />
+                    <select name="sheet"></select>
+                    <br /><br />
+                </div>
+
+				<div class="disp">
+                    Display 
+                    <br />
+                    <select name="display">
+                    <option value="0">Default</option>
+                    <option value="1">Percentage</option>
+                    </select>
+                    <br /><br />
+                </div>
+
+				<div class="localizable"><input type="checkbox" name="localizable"/> Localizable<br /><br /></div>
+
+				<div class="custom">
+                    Type
+                    <select name="ctype"></select>
+                    <br /><br />
+                </div>
+				<div class="opt"><input type="checkbox" name="req"/> Required</div>
+			</div>
+            <br /><br />
+			<p class="buttons">
+				<input class="edit" type="submit" value="Modify" id="editBtn" />
+				<input class="create" type="submit" value="Create" id="createBtn" />
+				<input type="submit" value="Cancel" id="cancelBtn" />
+			</p>
+
+			</form>').appendTo(contentModal);
+
+        var sheets = form.find("[name=sheet]");
+		sheets.empty();
+		for( i in 0...base.sheets.length ) {
+			var s = base.sheets[i];
```

Comment:
Mix of tabs and spaces. Set vscode setting
"files.trimTrailingWhitespace": true,
 "editor.insertSpaces": false,